### PR TITLE
refactor: Use new fancy anstyle API

### PR DESCRIPTION
**Pull request by hi-rustin**
1. Used the new fancy anstyle API
2. Verified terminal styling in some tests.

### CODE REVIEW

File: tests/testsuite/cargo_information/help/stdout.term.svg
Comment (hi-rustin) [inline tests/testsuite/cargo_information/help/stdout.term.svg:1]: I think we have some bugs here. We cannot render the color for help correctly.

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

File: src/ops/view.rs [hunk 1]
```diff
@@ -45,22 +48,22 @@ pub(super) fn pretty_view(
         (Some(latest), false) if latest.as_summary().version() != package_id.version() => {
             write!(
                 stdout,
-                " {warn}(latest {} {note}from {}{warn}){reset}",
+                " {warn}(latest {} {note}from {}{warn}){warn:#}",
```
Comment (hi-rustin) [inline src/ops/view.rs:51]: I am not sure if this is correct. Do we always need to reset first before we change color?

Comment (epage) [reply src/ops/view.rs:51]: Yes, we do.

File: tests/testsuite/cargo_information/help/stdout.term.svg
Comment (epage) [reply tests/testsuite/cargo_information/help/stdout.term.svg:1]: Cargo had to explicitly do work to test the color of `--help`

File: src/ops/view.rs (hunk 1 above)
Comment (hi-rustin) [reply src/ops/view.rs]: Thanks!

Comment (epage) [reply src/ops/view.rs]: You need to close each style before opening the next.

Comment (hi-rustin) [reply src/ops/view.rs]: Yes. I think I've updated it.

**Review by epage — APPROVED** [review]

File: tests/testsuite/cargo_information/help/stdout.term.svg
Comment (epage) [reply tests/testsuite/cargo_information/help/stdout.term.svg:1]: imo this can move forward without testing of colors of `--help`.

**Review by hi-rustin — COMMENTED** [review]
🔢 Self-check

**Pull request merged**